ee/cron/chunksHostLoads.js

```diff
@@ -0,0 +1,189 @@
+// @ts-check
+const AWS = require('aws-sdk');
+const { callbackify } = require('util');
+
+const config = require('../../lib/config');
+
+// These are used as IDs when reading CloudWatch metrics. They must start with
+// a lowercase letter and contain only numbers, letters, and underscores.
+const PAGE_VIEWS_PER_SECOND = 'pageViewsPerSecond';
+const ACTIVE_WORKERS_PER_SECOND = 'activeWorkersPerSecond';
+const LOAD_BALANCER_REQUESTS_PER_MINUTE = 'loadBalancerRequestsPerMinute';
+
+/**
+ * Finds the maximum value in the given array.
+ * @param {number[]} arr
+ */
+function arrayMax(arr) {
+  return arr.reduce((a, b) => Math.max(a, b), 0);
+}
+
+module.exports.run = callbackify(async () => {
+  if (
+    !config.runningInEc2 ||
+    !config.chunksLoadBalancerDimensionName ||
+    !config.chunksTargetGroupDimensionName
+  ) {
+    return;
+  }
+
+  const now = Date.now();
+  const cloudwatch = new AWS.CloudWatch();
+
+  const metrics = await cloudwatch
+    .getMetricData({
+      // Use the last 15 minutes of data.
+      // TODO: make this configurable?
```

Comment:
👀

Reply:
Yes please! The equivalent of https://github.com/PrairieLearn/PrairieLearn/blob/5526b70f2792354bdd4c52cc2cd3ce2bac0e035f/lib/config.js#L108

Reply:
Added in [96ac1ce](https://github.com/PrairieLearn/PrairieLearn/pull/6652/commits/96ac1ce0c7073339f5260861308ed04fe2ed6771)! The config option is named `chunksHostAutoScalingHistoryIntervalSec`, seem reasonable? (Note that I also renamed the cron job from `chunksHostLoads` to `chunksHostAutoScaling`.)

Reply:
Sounds good to me